src/domain/landingPage/LandingPagePreview.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
is there need to change this import style?

Reply:
Typescripts complains about it on intelliJ, I think that's more proper way to import.

Reply:
in tsconfig we have

`"esModuleInterop": true`

so we can import "normally"